Project/Engine/src/sysFrame/System/Json/Json.cpp

```
@@ -29,56 +31,79 @@ void Json::LoadJson(const std::string& _path) {
     auto& data = datas_[_path];
 
-    Object object;
+    Object& object = data[_group];
     object[_key] = _value;
+}
 
-    Group group;
-	group[_group] = object;
-
-    data.push_back(group);
+Json::Group Json::GetValueArray(const std::string& _path) {
```

Cppcheck (2.10-2)

[style] The function 'GetValueArray' is never used.

(unusedFunction)